extensions/amp-viewer-integration/0.1/messaging/messaging.js

```diff
@@ -116,16 +107,101 @@ export class WindowPortEmulator {
  * ampdoc and Bob is the viewer.
  */
 export class Messaging {
+  /**
+   * Performs a handshake and initializes messaging.
+   * @param {!Window} target - window containing AMP document to perform handshake with
+   * @param {?string=} opt_token - message token to verify on incoming messages (must be provided as viewer parameter)
+   * @param {?number=} opt_interval - how often to attempt handshake (in ms)
```

Comment:
I'd rather not have as an argument. Very unlikely a viewer can pick a good value that we can't otherwise predict here.

Reply:
Removed argument.